### DISCUSSION TITLE
Support localization for `ui.aggrid`

### DISCUSSION THREAD

**Discussion by dorel14** [zauberzeug/nicegui#3899] [Ideas / Feature Requests]
Hello 

Is it possible to make us able to set up Localisation in aggrid 

Here is the doc [https://www.ag-grid.com/javascript-data-grid/localisation/](https://www.ag-grid.com/javascript-data-grid/localisation/)

Thx

**Comment by falkoschindler** [comment]
Hi @dorel14,

In principle localization already works:
```py
ui.aggrid({
    'columnDefs': [
        {'headerName': 'Name', 'field': 'name'},
        {'headerName': 'Age', 'field': 'age'},
    ],
    'rowData': [],
    'localeText': {
        'noRowsToShow': 'Keine Zeilen',
    },
})
```
<img width="480" alt="Screenshot 2024-10-19 at 19 47 39" src="https://github.com/user-attachments/assets/90e18084-8f0d-4009-88a4-8f887227ea21">

Or you load the official AG Grid localization:
```py
URL = 'https://cdn.jsdelivr.net/npm/@ag-grid-community/locale@32.2.2/dist/umd/@ag-grid-community/locale.min.js'
ui.add_head_html(f'<script src="{URL}"></script>')

ui.aggrid({
    'columnDefs': [
        {'headerName': 'Name', 'field': 'name'},
        {'headerName': 'Age', 'field': 'age'},
    ],
    'rowData': [],
    ':localeText': 'AG_GRID_LOCALE_DE',
})
```

Sure, we could include it in the NiceGUI package. But it would add about another megabyte of package size.
